shared/Docs/Layout.tsx

```diff
@@ -98,11 +101,25 @@ export function Layout({
               <Header />
               <Navigation className="hidden lg:mt-6 lg:block" />
             </motion.header>
-            <div className="relative px-4 pt-14 sm:px-6 lg:px-8">
-              <main className="py-16">
+
+            {hidePageSidebar ? null : (
+              <motion.header
+                layoutScroll
+                className="fixed inset-y-0 my-14 pt-16 pb-12 right-0 z-40 hidden w-60 border-l border-slate-900/10 px-6 2xl:px-10 dark:border-white/10 xl:block 2xl:w-96"
+              >
+                <div className="pt-2">
+                  <PageSidebar />
+                </div>
+              </motion.header>
+            )}
+
+            <div className="relative px-4 pt-14 sm:px-6 lg:px-8 xl:pl-8 xl:pr-16 xl:mr-32 2xl:mr-10">
+              <main className="py-16 pr-20">
```

Comment:
Can we reduce and/or remove the right padding on small screens. It makes smaller screens not use the full width when the nav sidebar is hidden

This PR:
<img width="644" alt="Screen Shot 2024-03-11 at 5 08 34 PM" src="https://github.com/inngest/website/assets/1509457/373e3789-91bf-4b74-8bf2-c12560b77feb">

Main branch:
<img width="540" alt="Screen Shot 2024-03-11 at 5 08 49 PM" src="https://github.com/inngest/website/assets/1509457/8126ec83-e31a-4932-adfc-6668a1df518d">